aws/resource_aws_lambda_event_source_mapping_test.go

```diff
@@ -246,35 +246,7 @@ func TestAccAWSLambdaEventSourceMapping_SQSBatchWindow(t *testing.T) {
 	})
 }
 
-func TestAccAWSLambdaEventSourceMapping_kinesis_disappears(t *testing.T) {
```

Comment:
This test duplicates the functionality of the below test (`testAccCheckAWSLambdaEventSourceMappingDisappears`), which is why this was consolidated.